cloud/aws/modules/ecs_fargate_service/main.tf

```diff
@@ -0,0 +1,312 @@
+# This is a collection of resources that we used to rely on
+# cn-terraform/ecs-fargate-service/aws to provide, but this module
+# frequently broke us, so we include the necessary resources here instead.
+# The resources have been streamlined for our purposes.
+#
+# In places where we were not defining variables, the defaults from
+# https://github.com/cn-terraform/terraform-aws-ecs-fargate-service/blob/a1c2ea9fddb0ce682d6dac2e9af6b615a994e494/variables.tf
+# were used.
+
+### ecs-alb replacement
+#------------------------------------------------------------------------------
+# S3 BUCKET - For access logs
+#------------------------------------------------------------------------------
+data "aws_elb_service_account" "default" {}
+
+locals {
+  # While we don't really need to do this, it keeps resource names consistent
+  # with previous deploys to avoid destroying and recreating a bunch of
+  # resources.
+  name_prefix = "${var.app_prefix}-civiform"
+}
+
+#------------------------------------------------------------------------------
+# APPLICATION LOAD BALANCER
+#------------------------------------------------------------------------------
+resource "aws_lb" "civiform_lb" {
+  name = substr("${local.name_prefix}-lb", 0, 31)
+
+  internal                         = false
+  load_balancer_type               = "application"
+  drop_invalid_header_fields       = false
+  subnets                          = var.public_subnets
+  idle_timeout                     = 60
```

Comment:
**thought (non-blocking):** will be good to be able to parameterize this in the future. cc @dkatzz since it may be relevant for performance work